app/helpers/foreman_ansible/ansible_roles_helper.rb

```diff
@@ -1,9 +1,21 @@
 module ForemanAnsible
   # Small convenience to list all roles in the UI
   module AnsibleRolesHelper
-    def ansible_roles
-      ForemanAnsible::RolesImporter.import!
-      AnsibleRole.all
+    def ansible_proxy_links(hash, classes = nil)
+      links = SmartProxy.with_features('Ansible').map do |proxy|
+        display_link_if_authorized(_('Import from %s') % proxy.name,
+                                   hash.merge(:proxy => proxy),
+                                   :class => classes)
+      end.flatten
+      links.unshift display_link_if_authorized(_('Import from localhost'), hash, :class => classes)
```

Comment:
`Foreman host` instead.. otherwise it's kind of confusing especially if you have a localhost proxy